.github/workflows/docs.yaml

```diff
@@ -0,0 +1,29 @@
+name: Documentation
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master]
+    paths: ['docs/**']
```

Comment:
it compiles docs from `click` too right? We should probably have it update on every push

Reply:
correct, maybe this should just be on release. otherwise, docs will show unreleased changes.

Reply:
👍 I think thats a good idea, can github version the docs?

Reply:
looks like there's a [plugin](https://github.com/jimporter/mike) to mkdocs to allow gh-pages to host multiple versions. for now, i think just having docs for the latest release version is sufficient. the main documentation is for the cli, for which there's no reason to always use the latest version